crates/swift-integration-tests/src/vec.rs

```diff
@@ -25,6 +25,28 @@ mod ffi {
             arg: Vec<TransparentEnumInsideVecT>,
         ) -> Vec<TransparentEnumInsideVecT>;
     }
+
+    extern "Rust" {
+        fn run_vec_tests();
```

Comment:
We need to call this on the Swift side in `VecTests.rs`

Reply:
https://github.com/chinedufn/swift-bridge/blob/master/SwiftRustIntegrationTestRunner/SwiftRustIntegrationTestRunnerTests/VecTests.swift#L11

something like:

```swift
/// Verify that Rust can pass `RustVec`s to and receive `RustVec`s from Swift.
func testRustCallsSwiftRustVecFunctions() {
    run_vec_tests()
}
```